scripts/travis_install

```diff
@@ -25,7 +25,7 @@ conda install --yes $DEPS_TRAVIS
 
 conda config --add channels bokeh
 
-CONDA_PY="${TRAVIS_PYTHON_VERSION/./}" conda build --quiet --no-test conda.recipe
+CONDA_PY="${TRAVIS_PYTHON_VERSION/./}" conda build --skip-existing --quiet --no-test conda.recipe
```

Comment:
This was to work around a bug in conda. But I fixed that, and I think the fix made it to the latest release. Will try removing it again.